packages/frontend/src/providers/TrackingProvider.tsx

```diff
@@ -210,6 +210,7 @@ export const TrackingProvider: FC<TrackingData> = ({
         ({ id, traits }: IdentifyData) => {
             if (mode && mode !== LightdashMode.DEMO) {
                 rudderAnalytics?.identify(id, traits, lightdashContext);
+                (window as any).$chatwoot.setUser(id, traits);
```

Comment:
```suggestion
                (window as any).$chatwoot?.setUser(id, traits);
```

Reply:
Thanks, this line shouldn't be in the PR. I moved the logic to App Provider